pkg/kwokctl/cmd/logs/logs.go

```diff
@@ -60,10 +60,18 @@ func runE(ctx context.Context, logger logger.Logger, flags *flagpole, args []str
 		return err
 	}
 
-	if flags.Follow {
-		err = rt.LogsFollow(ctx, args[0], os.Stdout)
+	if args[0] == "audit" {
```

Comment:
any particular reason to treat "audit" particularlly? (instead of handling it in general `rt.Logs()`?

Reply:
At `Docker` runtime, the logs are obtained by calling the `docker logs` command
`audit` is an additional audit log file written by apiserver and fetched by `kwokctl` using `tail -f`.

`Kind` runtime does the same